### PR TITLE
fix(ENTESB-13200): Sort out shortcomings in templates

### PR DESCRIPTION
**Pull request by phantomjinx**

### CODE REVIEW

**Comment by heiko-braun** [comment]
That’s a good  start, but we need to restore the param IMAGESTREAM_NAMESPACE and corresponding imagestreams as well. On the eval cluster, the images are imported manually and referenced from a different namespace.

File: templates/fuse-online-template.yml [hunk 1]
```diff
@@ -607,7 +907,7 @@ objects:
               secretKeyRef:
                 key: OPENSHIFT_OAUTH_CLIENT_SECRET
                 name: syndesis-global-config
-          image: registry.redhat.io/openshift4/ose-oauth-proxy:4.2
+          image: quay.io/openshift/origin-oauth-proxy:v4.0.0
```
Comment (phantomjinx) [inline templates/fuse-online-template.yml]: @heiko-braun @lgarciaaco 
Is this a legitimate difference between 1.8 and 1.9? Used the upstream master to generate this now (after modifying and fixing errors) and this is the image value kicked out from the [config.yml](https://github.com/syndesisio/syndesis/blob/master/install/operator/build/conf/config.yaml#L38) and this is indeed the same as in the product repo too.

File: templates/fuse-online-template.yml [hunk 2]
```diff
@@ -1411,7 +1712,7 @@ objects:
         - args:
           - --config.file=/etc/prometheus/prometheus.yml
           - --storage.tsdb.retention=30d
-          image: registry.redhat.io/openshift3/prometheus:v3.9
+          image: docker.io/prom/prometheus:v2.1.0
```
Comment (phantomjinx) [inline templates/fuse-online-template.yml]: Same question for this one. See above.

**Comment by phantomjinx** [comment]
> That’s a good start, but we need to restore the param IMAGESTREAM_NAMESPACE and corresponding imagestreams as well. On the eval cluster, the images are imported manually and referenced from a different namespace.

The images are definitely tagged with the image namespace, eg. fuse-ignite, after the perl import script is executed. However, that can be encompassed by setting the [SYNDESIS_REGISTRY](https://github.com/syndesisio/fuse-online-install/blob/8c145ac5abd1f1c6732c48252d215de6f2b07318/templates/fuse-online-template.yml#L550) parameter to '<my-registry-hostname>:<port>/fuse-ignite'.

We can get some of the ImageStreams back using the `operator install forge` command (obviously can add them manually) but I note the syndesis-db one has disappeared from the operator code entirely.

So I think perhaps a collaborative coding session going through a diff of 1.8 and 1.9 might be useful to address some of these differences.

**Comment by heiko-braun** [comment]
Superseded by #231

**Pull request closed without merging**